CIP-0146/README.md

```diff
@@ -0,0 +1,220 @@
+---
+CIP: 146
+Title: Multi-signatures wallet registration and discovery
+Category: Wallets
+Status: Proposed
+Authors:
+  - Ola Ahlman <ola.ahlman@tastenkunst.io>
+  - Marcel Baumberg <marcel.baumberg@tastenkunst.io>
+Implementors:
+  - Eternl <https://eternl.io/>
+Discussions:
+Created: 2025-01-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This document describes how to format and register a multi-signatures wallet registration on the blockchain. The 
+individual parties that are part of the multi-signatures wallet can through this registration transaction easily be 
+discovered and added to the Cardano wallet software that implement the standard.
+
+This standard both extend on and add restrictions to
+[CIP-1854 | Multi-signatures HD Wallets](https://github.com/cardano-foundation/CIPs/tree/master/CIP-1854/README.md)
+to provide the structure of transaction auxiliary data.
+
+## Motivation: why is this CIP necessary?
+
+Term     | Definition
+---      | ---
+Multisig | Shorthand for Multi-party signature scheme.
+CSL      | [Cardano serialization lib](https://github.com/Emurgo/cardano-serialization-lib) by Emurgo.
+
+Multisig wallets have the ability to communicate scripts and metadata ahead of time through a registration 
+transaction utilizing the transaction auxiliary data. This is a convenient way for the multisig participants to 
+discover the multisig wallet before it's first usage.
+
+A common structure of the auxiliary data is needed for interoperability between wallets and web applications. The 
+metadatum label **1854** is used to define the native script types added in the auxiliary data and optionally provide 
+information about the wallet and its participants. 
+
+## Specification
+
+The following rules apply for a multisig registration to be valid:
+
+- Auxiliary data with a non-empty native scripts array.
+- Auxiliary data with label 1854 metadata that at a minimum includes a mapping for script types.

```

Comment:
```suggestion
- Auxiliary data with label `1854` metadata that at a minimum includes a mapping for script types.
```

Reply:
Datum being the singular form and data plural, in this case it's a reference to the whole metadata object and not the label or a single value. I could reword it to below if that's preferable?
> metadatum label `1854` metadata.

Reply:
@Scitz0 I see what you mean.  In that case `metadata` makes sense because... like most popular uses of the word... it's an arrangement composed ultimately of `metadatum`s 😧 and therefore redundant & not necessary to use both the singular & plural.  TL;DR so I think the way you originally wrote it is fine.

Reply:
Appears resolved in subsequent commits.